versioned_docs/version-5.x/redux-integration.md

```diff
@@ -85,6 +85,9 @@ function StaticCounter({ route }) {
 
 This is not possible. We don't support it because it's too easy to shoot yourself in the foot and slow down / break your app.
 
-However it's possible to use [`redux-devtools-extension`](https://github.com/zalmoxisus/redux-devtools-extension) to inspect the navigation state and actions, as well as perform time travel debugging. This works out of the box without any extra configuration.
+However it's possible to use [`redux-devtools-extension`](https://github.com/zalmoxisus/redux-devtools-extension) to inspect the navigation state and actions, as well as perform time travel debugging.
 
 If you use [`react-native-debugger`](https://github.com/jhen0409/react-native-debugger), it already includes this extension.
+
+You have to enable it by including the following line in your code :
+ `global.REACT_NAVIGATION_REDUX_DEVTOOLS_EXTENSION_INTEGRATION_ENABLED = true;`
```

Comment:
Where should one at best place this line?

Reply:
I put it into the index.js at the root of your project .
```
if (__DEV__) {
  global.REACT_NAVIGATION_REDUX_DEVTOOLS_EXTENSION_INTEGRATION_ENABLED = true
  AppRegistry.registerComponent('NameOfApp', () => AppDebug)
} else {
  AppRegistry.registerComponent('NameOfApp', () => App)
}
```